packages/compass-aggregations/src/constants.js

```diff
@@ -92,14 +92,18 @@ export const STAGE_SPRINKLE_MAPPINGS = {
     link: 'https://docs.mongodb.com/manual/reference/operator/aggregation/currentOp/#pipe._S_currentOp',
     tooltip: ''
   },
-  $facet: {
-    link: 'https://docs.mongodb.com/manual/reference/operator/aggregation/facet/#pipe._S_facet',
-    tooltip: 'Allows for multiple parellel aggregations to be specified.'
+  $densify: {
+    link: 'https://docs.mongodb.com/rapid/reference/operator/aggregation/densify',
+    tooltip: 'Creates new documents in a sequence of documents where certain values in a field are missing.'
   },
   $documents: {
     link: 'https://docs.mongodb.com/v5.1/reference/operator/aggregation/documents/#mongodb-pipeline-pipe.-documents',
     tooltip: 'Returns literal documents from input values.'
   },
+  $facet: {
+    link: 'https://docs.mongodb.com/manual/reference/operator/aggregation/facet/#pipe._S_facet',
```

Comment:
```suggestion
    link: 'https://docs.mongodb.com/manual/reference/operator/aggregation/facet/',
```

(it doesn’t look like that anchor exists on the page?)

Reply:
I haven't added facet in this pr, just moved it in code. And also I can see that almost all stages has this postfix. Do you know why we use it or why not should be using it?

Reply:
Alright, then let's leave it :) I don't know why we have those anchors, I assume at some point they had some relevance that has now disappeared.